app/controllers/users_controller.rb

```diff
@@ -34,6 +45,7 @@ def log_in
   
   private
   def user_params
-    params.require(:user).permit(:email, :password_digest, :shelters)
+    # params.require(:user).permit(:email, :password_digest)
+    params.permit(:email, :password, :password_confirmation)
```

Comment:
still confused on if we need password confirmation in this strong params since we have the logic for it and its not actually a column on the users table? but if it works i'm cool with it